src/assets/stylesheets/containers/experience.sass

```diff
@@ -0,0 +1,111 @@
+.heading
+  font-size: $font-size-xl * 2
+
+  &--primary-branded
+    @extend .heading
+    &::before
+      @extend .divider
+      content: ' '
+      display: block
+      margin: 0 auto $m-xl auto
+
+  &--secondary-branded
+    @extend .heading
+    &::before
+      @extend .divider
+      content: ' '
+      display: block
+      margin:
+        top: $m-xl
+        bottom: $m-xl
+
+.column-section
+  margin:
+    left: $m-xxl * 2
+    right: $m-xxl * 2
+  display: flex
+  justify-content: space-around
+  align-items: flex-start
+
+  &__column
+    flex: 1 1 33%
+    display: flex
+    justify-content: center
+    flex-direction: column
+    &:not(:last-child)
+      margin-right: $m-lg
+
+  &__description
+    font-size: $font-size-lg * 1.1
+    color: rgba($color-gray, 0.5)
+    padding:
+      bottom: $m-xl
+    margin:
+      top: $m-md
+      bottom: $m-xl
+    border:
+      bottom: $m-xs/8 solid rgba($color-gray, 0.3)
+
+.card
+  background-color: $color-white
+  &--visible-on-top
+    @extend .card
+    position: relative
+    z-index: $z-higher
+    box-shadow: -$m-lg $m-lg $m-xl rgba($color-violet, 0.2)
+
+  &:last-child:nth-child(n+2)
+    margin-top: $m-xl
+    position: relative
+    left: 55px
+
+  &__subcaption
+    color: $color-gray
+    font-size: $font-size-sm
+    padding: $m-sm
+    margin: $m-sm
+    text-transform: uppercase
+    border-bottom: $m-xs/4 solid $color-gray
+    display: inline-block
+
+  &__caption
+    font-weight: bold
+    font-size: $font-size-lg * 1.3
+    padding:
+      left: $m-sm
+      bottom: $m-sm
+
+.divider
+  border: $m-xs/8 solid $color-violet
+  border-radius: 7px 7px 7px 7px
+  background-color: $color-violet
+  width: $m-xxl
+  height: $m-md / 1.9
+
+.link-with-icon
+  font-size: $font-size-base * 1.1
+  color: rgba($color-gray, 0.5)
+  display: inline-block
+  position: relative
+  text-decoration: none
+  padding:
+    top: $m-sm
+    bottom: $m-sm
+  left: $m-xxl + $m-sm
+  &::before
+    position: absolute
+    right: 100%
+    top: 0
+    content: '►'
```

Comment:
Use real svg icons instead of unicode symbols, which could be easily overridden on mobile devices